### PR TITLE
  Add support for Firefox while leaving compatibility for Chrome

### PR DESCRIPTION
**Pull request by MatejMecka**
This PR allows the user to load the SwagLyrics Extension into Firefox browsers allowing for a larger userbase

### CODE REVIEW

**Comment by JamesVStone** [comment]
@aadibajpai works for me when loading the extension in temporary add-on mode.

File: bridge.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ let cont = false;
 const selectors = {
   albumArt:
     '.cover-art-image',
-  trackName: 'a[data-testid="nowplaying-track-link"]',
+  trackName: 'div.now-playing-bar__left div.now-playing div._2d35c1726829c507fca5a9b5b1aae1a2-scss.ellipsis-one-line div.c319b99793755cc3bba709fe1b1fda42-scss.ellipsis-one-line div.react-contextmenu-wrapper span a',
```
Comment (JamesVStone) [inline bridge.js:11]: Have you tested this expanded selector on Chrome @MatejMecka

Comment (MatejMecka) [reply bridge.js:11]: Yes I have, it worked for me

**Comment by JamesVStone** [comment]
When you install an addons in temporary mode it only keeps it until I restart firefox. Some posts I have checked on MDN say it means there is some problem with `manifest.json`. Hopefully it will be an easy fix, then Aadi will be able to get it signed and uploaded to the firefox store.

source: https://support.mozilla.org/en-US/questions/1158221

**Review by aadibajpai — APPROVED** [review]
gg thanks, sorry for the delay

**Pull request merged**